package.json

```diff
@@ -41,6 +41,7 @@
     "react": "18.2.0",
     "react-i18next": "^11.16.7",
     "react-native": "0.68.1",
+    "react-native-contacts": "^7.0.6",
```

Comment:
Normally you should not need this dependency here as the package has been added to "dependencies" of core package, only "peerDependencies" should be given to both core package and global package. 
If it's not working when removing this dependency here, please move "react-native-contacts" from dependencies to peerDependencies in core package.